x/auth/ante/fee.go

```diff
@@ -54,9 +53,12 @@ func (dfd DeductFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bo
 		err      error
 	)
 
```

Comment:
When inspecting the changes made in the AnteHandler in old Heimdall, we see this code was added:

		//Check whether the chain has reached the hard fork length to execute milestone msgs
		if ctx.BlockHeight() < helper.GetAalborgHardForkHeight() && (stdTx.Msg.Type() == checkpointTypes.EventTypeMilestone || stdTx.Msg.Type() == checkpointTypes.EventTypeMilestoneTimeout) {
			newCtx = SetGasMeter(simulate, ctx, 0)
			return newCtx, sdk.ErrTxDecode("error decoding transaction").Result(), true
		}

Should it also be part of the `AnteHandle` method in this version?

Reply:
Not sure if we will need it as on the new chain we won't have any previous HF, hence milestones will be enabled by default. Ported this code anyway with a comment to eventually enable it

Reply:
Resolved